sdk/nodejs/automation/cmd.ts

```diff
@@ -166,7 +166,7 @@ export class PulumiCommand {
         if (path.isAbsolute(this.command)) {
             const pulumiBin = path.dirname(this.command);
             const sep = os.platform() === "win32" ? ";" : ":";
-            const envPath = pulumiBin + sep + (additionalEnv["PATH"] || process.env["PATH"]);
+            const envPath = pulumiBin + sep + (additionalEnv["PATH"] || process.env.PATH);
```

Comment:
This is documented here as being case-insensitive.  (I don't know if it's the same for the `process.env["PATH"]` version?) https://github.com/nodejs/node/commit/0a8d0eaa9be35bfa899135118f1917dbd57dbda1

I think we can reasonably expect `additionalEnv["PATH"]` to always be uppercase.